core/replay/replay.py

```diff
@@ -200,6 +208,27 @@ def main():
 
         logger.info(f'Exported system tables to {g_config["replay_output"]}')
 
+    # uploading replay logs to s3
+
+    bucket = bucket_dict(g_config["workload_location"])
+    object_key = 'replay_logs.zip'
+    zip_file_name = f'replay_{replay_id}.zip'
+    logger.info(f"Uploading replay logs to {bucket['bucket_name']}/{bucket['prefix']}")
+    import pdb;pdb.set_trace()
```

Comment:
I see the PR is in draft, but still want to add this comment as I spotted it.

Reply:
Thanks